microcosm_resourcesync/main.py

```diff
@@ -57,7 +57,6 @@ def sync(context, origins, destination, **kwargs):
         echo("Reading resources from: {}".format(origin), err=True)
         resources.extend(origin.read(**kwargs))
 
-    echo("Toposorting {} resources".format(len(resources)), err=True)
```

Comment:
I moved the echo to have a better count. I haven't debugged yet why we get duplicates in the first place.